packages/cashier/src/components/cashier-onboarding/cashier-onboarding-side-note.tsx

```diff
@@ -44,17 +44,39 @@ const CashierOnboardingSideNote = observer(({ is_crypto }: TCashierOnboardingSid
         );
     };
 
-    return (
-        <div>
-            <Text className='cashier-onboarding-side-note__text' color='prominent' weight='bold' size='xs' as='p'>
-                {is_crypto ? (
-                    <Localize i18n_default_text='This is your {{currency_code}} account.' values={{ currency_code }} />
-                ) : (
+    const getHeaderTitle = () => {
+        if (!is_pre_appstore) {
+            if (!is_crypto)
```

Comment:
Flip the check to `is_crypto` for better code readability!

Reply:
Done 😃 👍🏼